### PR TITLE
9847 Applying editorial changes

### PR DESCRIPTION
**Pull request by andymunro**
@stianst I made changes along the following lines:
* Change to avoid first person plural (we) which is inconsistent with Keycloak style
* Removed contractions
* Changed task sections to be gerunds and concept sections to nouns.
* Made the text more concise.

### CODE REVIEW

File: docs/guides/src/main/server/containers.adoc [hunk 1]
```diff
@@ -4,18 +4,18 @@
...
 priority=20
 includedOptions="db db-url db-username db-password features hostname https-key-store-file https-key-store-password metrics-enabled">
 
-Keycloak is handling containerized environments like Kubernetes or OpenShift as first-class-citizens. In this guide you'll learn how to run and optimize the Keycloak container image to have the best experience running a Keycloak container.
+Keycloak handles containerized environments such as Kubernetes or OpenShift as first-class citizens. This guide describes how to optimize and run the Keycloak container image to provide the best experience running a Keycloak container.
 
-== Create an optimized container image
-To get the best startup experience for your Keycloak container, we recommend building an optimized container image by running the `build` step explicitly before starting:
+== Creating an optimized container image
+For the best start up of your Keycloak container, build an optimized container image by running the `build` step immediately before starting.
```
Comment (DGuhr) [inline docs/guides/src/main/server/containers.adoc:15]: Process-wise, this doesn't have to happen immediately before starting, it just has to happen any time before you start it. Suggestion: remove "immediately" completely.

File: docs/guides/src/main/server/containers.adoc [hunk 2]
```diff
@@ -100,8 +109,10 @@ podman|docker run --name keycloak_auto_build -p 8080:8080 \
         --https-key-store-file=<file> --https-key-store-password=<password>
 ----
 
-Running the above will spin up a Keycloak server which detects and applies the build configuration first. In the example, it's  `--db=postgres --features=token-exchange` to set the used database vendor to postgres and enable the token exchange feature.
+Running this command starts a Keycloak server that detects and applies the build configuration first.
+In the example, the line  `--db=postgres --features=token-exchange` sets the database vendor to PostgreSQL and enables the token exchange feature.
 
-Keycloak will then start up and apply the configuration for the specific environment. This approach results in a significantly bigger startup time, and an image that is mutable, and is therefor not the best practice.
+Keycloak then starts up and applies the configuration for the specific environment.
+This approach significantly increases startup time and creates image that is mutable, which is not the best practice.
```
Comment (DGuhr) [inline docs/guides/src/main/server/containers.adoc:116]:
```suggestion
This approach significantly increases startup time and creates an image that is mutable, which is not the best practice.
```

File: docs/guides/src/main/server/containers.adoc (hunk 1 above)
Comment (DGuhr) [inline docs/guides/src/main/server/containers.adoc:15]:
```suggestion
For the best start up of your Keycloak container, build an optimized container image by running the `build` step before starting.
```

File: docs/guides/src/main/server/containers.adoc [hunk 3]
```diff
@@ -100,8 +109,10 @@ podman|docker run --name keycloak_auto_build -p 8080:8080 \
         --https-key-store-file=<file> --https-key-store-password=<password>
 ----
 
-Running the above will spin up a Keycloak server which detects and applies the build configuration first. In the example, it's  `--db=postgres --features=token-exchange` to set the used database vendor to postgres and enable the token exchange feature.
+Running this command starts a Keycloak server that detects and applies the build configuration first.
```
Comment (DGuhr) [inline docs/guides/src/main/server/containers.adoc:112]:
```suggestion
Running this command starts a Keycloak server that detects and applies the build options first.
```
See #9709 for details about the general naming. "Build configuration" might be a bit ambiguous here.

**Review by DGuhr — CHANGES_REQUESTED** [review]

**Review by DGuhr — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**